sql/moz-fx-data-shared-prod/mozilla_org_derived/downloads_with_attribution_v2/metadata.yaml

```
@@ -47,11 +47,11 @@ labels:
   incremental: true
   schedule: daily
   owner1: gleonard
-scheduling:
-  dag_name: bqetl_download_funnel_attribution
-  task_name: mozilla_org_derived__downloads_with_attribution__v2
-  date_partition_parameter: download_date
-  date_partition_offset: -1
+#scheduling:
+#  dag_name: bqetl_download_funnel_attribution
+#  task_name: mozilla_org_derived__downloads_with_attribution__v2
+#  date_partition_parameter: download_date
+#  date_partition_offset: -1
```

As you've noticed per #7507, removing all scheduling metadata for ETLs can cause problems (e.g. dryrun failures).  I'd recommend just commenting out the `dag_name` & `task_name` (if present) for these deprecated ETLs you want to unschedule.
```suggestion
scheduling:
  #dag_name: bqetl_download_funnel_attribution
  #task_name: mozilla_org_derived__downloads_with_attribution__v2
  date_partition_parameter: download_date
  date_partition_offset: -1
```
Or another solution could be updating `bqetl` to generally ignore/skip deprecated ETLs.